KubeArmor/core/kubeUpdate.go

```diff
@@ -456,36 +457,42 @@ func (dm *KubeArmorDaemon) WatchK8sPods() {
 				pod.Metadata["namespaceName"] = event.Object.ObjectMeta.Namespace
 				pod.Metadata["podName"] = event.Object.ObjectMeta.Name
 
+				pod.Labels = map[string]string{}
+				for k, v := range event.Object.Labels {
+					if k == "pod-template-generation" {
+						continue
+					}
+
+					if k == "controller-revision-hash" {
+						continue
+					}
+					pod.Labels[k] = v
```

Comment:
The labels that are added as part of pod.Labels are shipped in every alert/telemetry. Sending `pod-template-hash` in such telemetry events does not make sense.
Can you check with these changes if the pod-labels (in json format) now contains `pod-template-hash` as well?

Reply:
No worries, we only need it once anyways. Won't use this map :+1: